<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>311</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLCDNumber" name="lcdNumber">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>291</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="put100">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>100</string>
   </property>
  </widget>
  <widget class="QPushButton" name="put500">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>500</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buyCoffee">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>120</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>coffee(100)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buyTea">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>150</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>tea(150)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buyMilk">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>180</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>milk(200)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="reset">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>300</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
  <widget class="QPushButton" name="put1000">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>1000</string>
   </property>
  </widget>
  <widget class="QPushButton" name="put10">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>10</string>
   </property>
  </widget>
  <widget class="QPushButton" name="put50">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>50</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buyCola">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>210</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>cola(350)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buyHot6">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>240</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>hot6(800)</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
